src/components/MainSection.js

```diff
@@ -24,6 +27,11 @@ import Footer from './Footer'
      this.setState({ filter })
    }
 
+   orderByPriority = (is_descending) => {
+     const lambda = x => is_descending * -1 * x.priority
```

Comment:
this function is expecting a number, but is called with boolean arguments on Line